### ISSUE TITLE
v0.5.7.4 Windows 10 Defender issues. 

### ISSUE THREAD

**Issue by SnakeyHips** [Kaldaien/FAR#57]
I'm having trouble downloading v0.5.7.4. Every time I do, Windows Defender deletes it as it's seeing it as a Trojan. Here's an image which shows the info Defender gives about it: http://i.imgur.com/EDWIfr2.png

I'm using Windows 10 Pro 64-bit. Every other version of FAR downloads fine, just this one is having Defender issues.

**Comment by Kaldaien** [comment]
Unfortunately I don't know what I did to trigger this, so fixing it probably won't happen any time soon.

Other users have reported that it began at version 0.5.7.2. I can say that nothing of any significance has been added to FAR since 0.5.7.0 and if an earlier version works for you go ahead and use that until I can fix this.

**Comment by SnakeyHips** [comment]
Yeah sure I'll carry on using 0.5.7.3. Maybe Defender is just being a pain. If you need any more information from my end just let me know.

**Comment by SnakeyHips** [comment]
Just updating saying I'm not getting this issue with 6.0 💃
